bug: The named parameter 'newReminder' isn't defined.

The named parameter 'newReminder' isn't defined.
Try correcting the name to an existing named parameter's name, or defining a named parameter with the name 'newReminder'.

```dart
Launching lib\main.dart on CPH2735 in debug mode...
lib/app.dart:88:53: Error: No named parameter with the name 'newReminder'.
            builder: (context) => AddReminderScreen(newReminder: reminder),
                                                    ^^^^^^^^^^^
lib/features/reminders/add_reminder_screen.dart:8:9: Context: Found this candidate, but the arguments don't match.
  const AddReminderScreen({super.key, this.existingReminder});
        ^^^^^^^^^^^^^^^^^
Target kernel_snapshot_program failed: Exception

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\src\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 52s
```

--- a/mobile_app/lib/app.dart
+++ b/mobile_app/lib/app.dart
@@ -85,7 +85,7 @@ class MedTrackApp extends StatelessWidget {
               : null;
 
           return MaterialPageRoute(
-            builder: (context) => AddReminderScreen(newReminder: reminder),
+            builder: (context) => AddReminderScreen(existingReminder: reminder),
           );
         }
 

